### ISSUE TITLE
Error in calculating reported cost under cb_explore.

### ISSUE THREAD

**Issue by rajan-chari** [VowpalWabbit/vowpal_wabbit#1811]
#### Describe the bug
**Email excerpt from Marco:** 
[Code location](https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/vowpalwabbit/cb_explore.cc#L244)
 
The line is calculating the soft-average cost, i.e., the average cost of the policy using ec.pred.a_s[i].score; for each action:
 
loss += get_unbiased_cost(c.known_cost, c.pred_scores, i) * ec.pred.a_s[i].score;
 
The issue is that the i is 0-indexed, but it’s used in get_unbiased_cost against the 1-index action label.
